### PR TITLE
Interactive List

### PR DESCRIPTION
**Pull request by Mesi21**
- Add a new JavaScript file and import it as a module
- Add methods to the status updates (completed: true/false).
- Add event listener to the checkbox (change).
- Update items object's value for completed key upon user actions.
- Store the updated array of items in local storage

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -6,45 +6,73 @@ import '@fortawesome/fontawesome-free/js/brands';
...
-    completed: false,
-    index: 2,
-  },
-];
+class ToDo {
+  constructor() {
+    this.tasks = [
+      {
+        description: 'task1',
+        completed: false,
```
Comment (rloterh) [inline src/index.js:15]: - [x] Your _imported modules_ do not contain methods related to `status update`. Kindly ensure that your `index.js` has one new JavaScript file imported as a module, which should contain methods related to the `status updates` (completed: true / false).

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Mesi, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦 🚦  

## Good Points 👍  
- Descriptive Pull Request
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (Mesi21) [reply src/index.js:15]: I added the toggleCheck.js file with the corresponding function.

**Review by AshakaE — APPROVED** [review]
### Hi @Mesi21  🙋‍♀️, 

# STATUS APPROVED ✔
Thanks for making the requested change 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**